.ci/docker/build.sh

```diff
@@ -41,6 +45,9 @@ BUILD_DOCS=1
 # Copy requirements-lintrunner.txt from root to here
 cp ../../requirements-lintrunner.txt ./
 
+# Copy arm setup script from root to here
+cp -r ../../examples/arm/ ./arm
```

Comment:
BTW how do we trigger an image build if examples/arm changes?

Reply:
we only should need `setup.sh` and patches FWIW. But your question is valid.

Reply:
It's harder to copy the specific files than the entire dir :p